components/compliance-service/habitat/hooks/run

```diff
@@ -124,6 +124,9 @@ CONFIG="$CONFIG --event-endpoint {{event.sys.ip}}:{{event.cfg.port}}"
  {{~/if}}
  {{~/eachAlive}}
 
+ # Create the compliance config file
+CONFIG="$CONFIG --config {{pkg.svc_data_path}}/.compliance-service.toml"
```

Comment:
Adding a config file for backing up the state of the project update. This file will be located at `/hab/svc/compliance-service/data/.compliance-service.toml`